src/GovUk.Education.ExploreEducationStatistics.Data.Services/SubjectMetaService.cs

```diff
@@ -306,18 +254,32 @@ private TimePeriodsMetaViewModel GetTimePeriods(IQueryable<Observation> observat
             return locationAttributeNode.IsLeaf
                 ? new LocationAttributeViewModel
                 {
-                    Label = locationAttributeNode.Attribute?.Name ?? string.Empty,
-                    Value = locationAttributeNode.Attribute?.Code ?? string.Empty
+                    Label = locationAttributeNode.Attribute.Name ?? string.Empty,
+                    Value = locationAttributeNode.Attribute.Code ?? string.Empty
                 }
                 : new LocationAttributeViewModel
                 {
-                    Label = locationAttributeNode.Attribute?.Name ?? string.Empty,
-                    Level = locationAttributeNode.Attribute?.GetType().Name,
-                    Value = locationAttributeNode.Attribute?.Code ?? string.Empty,
-                    Options = locationAttributeNode.Children.Select(BuildLocationAttributeViewModel).ToList()
+                    Label = locationAttributeNode.Attribute.Name ?? string.Empty,
+                    Level = locationAttributeNode.Attribute.GetType().Name,
+                    Value = locationAttributeNode.Attribute.Code ?? string.Empty,
+                    Options = DeduplicateLocationViewModels(
+                            locationAttributeNode.Children.Select(BuildLocationAttributeViewModel)
+                        )
+                        .OrderBy(OrderLocationViewModel)
+                        .ToList()
                 };
         }
 
+        private static string OrderLocationViewModel(LocationAttributeViewModel option)
```

Comment:
Cool, thanks for sorting out the ordering! 🙂 I knew there was no ordering in the new stuff but didn't highlight it while it was turned off by default because we had upcoming tickets to deal with ordering. It will be cool if this satisfies all the ordering requirements.